pipeline/outputs/observe.md

```diff
@@ -10,13 +10,12 @@ The following are the specific HTTP parameters to employ:
 
 | Key                        | Description                                                                                                                                                                                                                                                                                                                        | default   |
 | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
-| host                       | IP address or hostname of Observe's data collection endpoint                                                                                                                                                                                                                                                                       | collect.observeinc.com |
+| host                       | IP address or hostname of Observe's data collection endpoint.  $(OBSERVE_CUSTOMER) is your [Customer ID](https://docs.observeinc.com/en/latest/content/common-topics/HelpfulHints.html?highlight=customer%20id#customer-id)                                                                                                                                                                                                                                                                      | ${OBSERVE_CUSTOMER}.collect.observeinc.com |
```

Comment:
So does this require an environment variable to be set for the default? I can imagine that might not be desirable in containers.

Reply:
Actually, it's meant to represent a token of sorts, to replace OBSERVER_CUSTOMER with a customer's Observe Customer ID when host is added to OUTPUTS.  Sorry for the confusion, does this make sense?

i.e. if my Observe customer id was "joe" then my host would now be https://joe.collect.observeinc.com

Reply:
Looks like that is what Grafana Cloud docs say: https://docs.fluentbit.io/manual/pipeline/outputs/loki#fluent-bit-+-grafana-cloud
Unfortunately the others seem a little more like real examples: https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch#aws-es

Reply:
@patrick-stephens yes, it's just a KEY.  I tried to make it obvious (and consistent with Observe documentation) by using OBSERVE_CUSTOMER.    If I updated the sample [OUTPUT] example to say:

host         my-observe-customer-id.collect.observeinc.com

Would this resolve this?

Reply:
Thanks @patrick-stephens !

Reply:
Can you remove the `${...}` syntax as that is the exact syntax used for variables?